client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala

```diff
@@ -1331,6 +1352,10 @@ class LifecycleManager(val appUniqueId: String, val conf: CelebornConf) extends
     appShuffleTrackerCallback = Some(callback)
   }
 
+  def registerAppShuffleDeterminate(appShuffleId: Int, determinate: Boolean): Unit = {
```

Comment:
Nit: `determinate` is an argument to the method, so shouldn't we just call this method `registerAppShuffle`

Reply:
Per my perspective, the whole point of this method is to pass determinate information of ShuffleMapStage to LifecycleManager, so it is a better name than registerAppShuffle
actually, it is handleGetShuffleIdForApp method that handles the job of "registerAppShuffle"